src/jquery.ime.inputmethods.js

```diff
@@ -1579,7 +1579,7 @@
 			inputmethods: [ 'my-mm3', 'my-xkb', 'my-shn' ]
 		},
 		shn: {
-			autonym: 'တႆး',
+			autonym: 'ၽႃႇသႃႇတႆး',
```

Comment:
It should probably be "လိၵ်ႈတႆး‎". That's what appears in Ethnologue and in language-data. 'ၽႃႇသႃႇတႆး' appears in Names.php, but I don't know where does it come from. I emailed the most prominent translator to ask about this.